var/spack/repos/builtin/packages/hwloc/package.py

```diff
@@ -57,6 +57,14 @@ class Hwloc(AutotoolsPackage):
         default=False,
         description='Enable the Cairo back-end of hwloc\'s lstopo command'
     )
+    variant(
+        'netloc',
+        default=False,
+        description="Enable netloc [requires MPI]"
+    )
+    
```

Comment:
Flake mentions that this line contains whitespace (as well as line 81)

Reply:
Fixed.